### ISSUE TITLE
Runtime exceptions (missing dependency?) on fresh build, OS X

### ISSUE THREAD

**Issue by awmcclain** [inkle/inky#112]
Hey all, I'm trying to build inky on OS X (sierra, 10.12.6) from a fresh clone.
Node v6.11.2, npm v3.10.10

After cloning and running `./BUILD_FOR_MAC.command`, and launching inky...
I can open an ink file, but nothing runs in the right panel, and I can't quit or open another ink file in the left panel.

I get this exception in the inky console after I launch:
```
Uncaught Error: Cannot find module 'fuzzaldrin-plus'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (/Volumes/Inky 1/Inky.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Volumes/Inky 1/Inky.app/Contents/Resources/app.asar/renderer/goto.js:5:31)
    at Object.<anonymous> (/Volumes/Inky 1/Inky.app/Contents/Resources/app.asar/renderer/goto.js:426:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```
Looks like it's trying to find the `fuzzaldrin` module but it can't?

Strange because I see fuzzaldrin-plus in app/node_modules/.
`RUN.command` gives the same runtime error.

**Comment by awmcclain** [comment]
`0.7.2` does not have this problem, FYI

**Comment by awmcclain** [comment]
@joethephish Ok! I figured out the issue -- fuzzaldrin-plus has to be compiled (running `grunt`) since all the lib files are in coffeescript, so when you get the package from your forked repo, inky looks for js files in `fuzzaldrin-plus/lib` but that directory hasn't been created.

I was able to fix the issue by running `grunt` in the fuzzaldrin module directory (which required an `npm install grunt grunt-contrib-coffee grunt-coffeelint`).

I don't have a lot of npm packaging experience either so I'm not sure if the best solution is to check in the compiled js files into your fork or to run a postinstall command to compile the module. Either way, right now, the `inky` repo is in a broken state.
